adapters/multiply_test.go

```diff
@@ -11,16 +12,24 @@ import (
 
 func TestMultiply_Perform(t *testing.T) {
 	tests := []struct {
-		name    string
-		times   int64
-		json    string
-		want    string
-		errored bool
+		name      string
+		params    string
+		json      string
+		want      string
+		errored   bool
+		jsonError bool
 	}{
-		{"string", 100, `{"value":"1.23"}`, "123", false},
-		{"integer", 100, `{"value":123}`, "12300", false},
-		{"float", 100, `{"value":1.23}`, "123", false},
-		{"object", 100, `{"value":{"foo":"bar"}}`, "", true},
+		{"string", `{"times":100}`, `{"value":"1.23"}`, "123", false, false},
+		{"integer", `{"times":100}`, `{"value":123}`, "12300", false, false},
+		{"float", `{"times":100}`, `{"value":1.23}`, "123", false, false},
```

Comment:
Can we add a negative number and 0 to the test suite? i.e. `"times": -5`

Reply:
Done.